[2023-02-22T16:12:03.931+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: csv_from_s3_to_s3_v01.test_connection manual__2023-02-22T16:12:02.168820+00:00 [queued]>
[2023-02-22T16:12:03.952+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: csv_from_s3_to_s3_v01.test_connection manual__2023-02-22T16:12:02.168820+00:00 [queued]>
[2023-02-22T16:12:03.952+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-22T16:12:03.953+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-02-22T16:12:03.953+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-22T16:12:03.964+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): test_connection> on 2023-02-22 16:12:02.168820+00:00
[2023-02-22T16:12:03.971+0000] {standard_task_runner.py:55} INFO - Started process 6960 to run task
[2023-02-22T16:12:03.982+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'csv_from_s3_to_s3_v01', 'test_connection', 'manual__2023-02-22T16:12:02.168820+00:00', '--job-id', '203', '--raw', '--subdir', 'DAGS_FOLDER/dag_cleaning_via_s3.py', '--cfg-path', '/tmp/tmpludm4ilg']
[2023-02-22T16:12:03.985+0000] {standard_task_runner.py:83} INFO - Job 203: Subtask test_connection
[2023-02-22T16:12:04.084+0000] {task_command.py:388} INFO - Running <TaskInstance: csv_from_s3_to_s3_v01.test_connection manual__2023-02-22T16:12:02.168820+00:00 [running]> on host 22976f44c0e9
[2023-02-22T16:12:04.177+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_from_s3_to_s3_v01
AIRFLOW_CTX_TASK_ID=test_connection
AIRFLOW_CTX_EXECUTION_DATE=2023-02-22T16:12:02.168820+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-22T16:12:02.168820+00:00
[2023-02-22T16:12:04.199+0000] {base.py:73} INFO - Using connection ID 'AWS' for task execution.
[2023-02-22T16:12:04.201+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='AWS', conn_type='aws') credentials retrieved from login and password.
[2023-02-22T16:12:05.164+0000] {python.py:177} INFO - Done. Returned value was: ['titanic.csv', 'tmp/']
[2023-02-22T16:12:05.226+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=csv_from_s3_to_s3_v01, task_id=test_connection, execution_date=20230222T161202, start_date=20230222T161203, end_date=20230222T161205
[2023-02-22T16:12:05.285+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-02-22T16:12:05.305+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
